### DISCUSSION TITLE
Develco meter not getting configured

### DISCUSSION THREAD

**Discussion by gtankit96** [Koenkk/zigbee2mqtt#11933] [Devices]
Develco meter (SMMZB-310) is not getting configured by zigbee2mqtt and giving following logs:-

Failed to configure develco energy meter (SMMZB-310) but was able to ping it , and this is the log message below:-

Zigbee2MQTT:error 2022-03-23 07:48:47: Failed to configure '0x0015bc001902abd3', attempt 1 (Error: ConfigureReporting 0x0015bc001902abd3/1 genOnOff([{"attribute":"onOff","minimumReportInterval":0,"maximumReportInterval":3600,"reportableChange":0}], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 6306 - 1 - 3 - 6 - 7 after 10000ms)
at Timeout._onTimeout (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
at listOnTimeout (internal/timers.js:557:17)
at processTimers (internal/timers.js:500:7))

Can anyone please help me here? @Koenkk

**Comment by Lokeshwarpandey17** [comment]
Provide proper configuration of Develco Meter in develco.js file and restart the meter and please share the logs

**Comment by derabbink** [comment]
I might be able to help here, if you could tell me how you purchased the device.
(I want one as well, but I cannot seem to find one. And once I have one I can likely provide a working config)
